frontend/src/views/CalculateNutrients/CalculateNutrients.tsx

```diff
@@ -75,19 +90,33 @@ export default function CalculateNutrients() {
   }, [activeField]);
 
   const balanceRow: CalculateNutrientsColumn = useMemo(() => {
-    const allRows = [
+    const cropAndOtherRows = [
       ...fieldList[activeField].Crops,
       ...fieldList[activeField].Fertilizers,
       ...fieldList[activeField].OtherNutrients,
     ];
+    const manureRows = fieldList[activeField].Nutrients.nutrientManures;
+
     return {
       name: 'Balance',
-      reqN: allRows.reduce((sum, row) => sum + (row.reqN ?? 0), 0),
-      reqP2o5: allRows.reduce((sum, row) => sum + (row.reqP2o5 ?? 0), 0),
-      reqK2o: allRows.reduce((sum, row) => sum + (row.reqK2o ?? 0), 0),
-      remN: allRows.reduce((sum, row) => sum + (row.remN ?? 0), 0),
-      remP2o5: allRows.reduce((sum, row) => sum + (row.remP2o5 ?? 0), 0),
-      remK2o: allRows.reduce((sum, row) => sum + (row.remK2o ?? 0), 0),
+      reqN:
```

Comment:
Having everything named the same also means not needing to handle the rows separately here.

Reply:
The point of renaming them was so you can do `...fieldList[activeField].Nutrients.nutrientManures` into allRows and keep the old lines here.